### DISCUSSION TITLE
Redundant User Icons in Editor

### DISCUSSION THREAD

**Discussion by ghost** [GitbookIO/community#335] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

<img width="570" alt="too-many-cooks" src="https://github.com/GitbookIO/community/assets/48954562/f1978c1c-95e4-4808-b8e5-f23821c7db18">

I am noticing a bunch redundant (i.e. icons representing the same user) being displayed across the top of the editor.

It appears that every time a logged in user accesses a page from "outside" (i.e. from a link pasted into a chat) a new icon appears. 

There are so many that often I have to hide the sidebar to expose the edit button.

Is this a feature?

**Comment by thebiglabasky** [comment]
Hi @dougscc I'm afraid that's not a feature but a bug. I filed it for the team to figure out and fix.
Thanks for flagging!

**Comment by aikoii** [comment]
Hey @dougscc, this issue should be solved 🤞. Let us know if you have trouble!

**Reply by ghost** [reply]
It appears to be so.
